src/FramesMonitor.js

```diff
@@ -72,8 +72,15 @@ class FramesMonitor extends EventEmitter {
 
         FramesMonitor._assertExecutable(ffprobePath);
 
-        this._config = _.cloneDeep(config);
-        this._url    = url;
+        this._config = {
+            ffprobePath,
+            bufferMaxLengthInBytes,
+            errorLevel,
+            exitProcessGuardTimeoutInMs,
+            timeout: timeoutInMs * 1000
```

Comment:
It looks like an error. You don't need to multiply by 1000.

Reply:
Also, why don't we use just `timeoutInMs`?

Reply:
FFprobe expects a  value in microseconds. That is why I make a one-time conversion.

> **rw_timeout**
 Maximum time to wait for (network) read/write operations to complete, in microseconds.